test/index.test.ts

```diff
@@ -85,7 +85,7 @@ it('test badge case - custom label', async () => {
   expect(str.indexOf(`<text x="50" y="138" textLength="715">${customLabel}</text>`) > 0).toBeTruthy();
 });
 
-it('test badge case - custom icon', async () => {
+it.skip('test badge case - custom icon', async () => {
```

Comment:
I'm not sure why this is failing, can anyone help debug this?

Reply:
@reubenjohn I fixed it. thanks

- https://github.com/badgen/badgen/pull/68